RELEASE-NOTES.txt

```diff
@@ -2,6 +2,7 @@
 
 16.5
 -----
+- [**] Shipping Labels: Fixed issue presenting the printing view for customs forms.
```

Comment:
np / ❓ : do we also need to add link to this PR? I'm not 100% sure if all release notes item need a link or not.